x-pack/platform/plugins/shared/fleet/server/types/rest_spec/agent.ts

```diff
@@ -100,7 +100,7 @@ export const MigrateOptionsSchema = {
   proxy_url: schema.maybe(schema.string()),
   staging: schema.maybe(schema.boolean()),
   tags: schema.maybe(schema.arrayOf(schema.string())),
-  replace_token: schema.maybe(schema.boolean()),
+  replace_token: schema.maybe(schema.string()),
```

Comment:
Should we keep this a string, so we get proper validation on the fleet API, backward compatibility, and the fix will be to convert what we sent the .fleet* index to string instead?